scripts/utils/settings.ts

```diff
@@ -62,3 +64,42 @@ export const eventSettings = new Map<string, EventSettings>([
 ]);
 
 export const elementsWithMissingEntrypoint = new Set<string>([]);
+
+export const descriptionOverrides = new Map<string, Partial<JSONSchemaForWebTypes>>([
```

Comment:
This was already fixed in the web components, see: https://github.com/vaadin/web-components/pull/5832

we still need this hotfix for the alpha9, but we can revert the release is done